### PR TITLE
PLANET-3486 Add and populate opt-ins custom meta box 

### PR DESCRIPTION
**Pull request by kirdia**
[Issue ](https://jira.greenpeace.org/browse/PLANET-3486)

### CODE REVIEW

File: classes/controller/class-enform-questions-list-table.php [hunk 1]
```diff
@@ -88,7 +96,7 @@ public function prepare_items() {
 	 * @return bool
 	 */
 	private function test_opt_in( $item ) {
```
Comment (sagarsdeshmukh) [inline classes/controller/class-enform-questions-list-table.php]: As we added a dynamic `type` variable in condition, I guess we need to change the function name and function doc desc.

File: classes/controller/class-enform-questions-list-table.php [hunk 2]
```diff
@@ -30,16 +30,24 @@ class Enform_Questions_List_Table extends \WP_List_Table {
 	 */
 	private $error;
 
+	/**
```
Comment (sagarsdeshmukh) [inline classes/controller/class-enform-questions-list-table.php]: Please add short description in doc comment (PHPCS!)

**Comment by sagarsdeshmukh** [comment]
I thought using one API call, we manage to serve the questions and opt-in data. I hope the extra call to EN API will not impact much.

**Comment by kirdia** [comment]
> I thought using one API call, we manage to serve the questions and opt-in data. I hope the extra call to EN API will not impact much.

Made the changes.
It won't impact too much, the second request will hit the cache

**Pull request merged**